.gitignore

```diff
@@ -2,3 +2,24 @@ out
 node_modules
 format.patch
 Makefile
+
+# dependencies
+/.pnp
+.pnp.js
```

Comment:
Can I ask what these are from?
And it would be nice if you can add other items why they are needed.
(I'm not certain about all the items here)

Reply:
This is the gitignore content that occurs automatically when you enter the command create-react-app.
In fact, it doesn't apply to our folder right now.
It seems no matter to erase, but it is probably considered a react recommendation.

Reply:
If they are not used, please remove them. We can add when we need.

Reply:
okay. Thank you for the feedback.